aexpect/client.py

```diff
@@ -1716,7 +1716,8 @@ def run_tail(
     encoding=None,
 ):
     """
-    Run a subprocess in the background and collect its output and exit status.
+    Runs the command, awaits for it to finish up to timeout and returns the
```

Comment:
The default for docstrings is to use imperative form just like commit message. So I think `Run` is still better than `Runs`.

Reply:
In fact it would even be better as "Run a command" instead of "Run the command" since the argument is a detail of the function and not the other way around.

Reply:
Run a command then wait for it to finish up to a given timeout...

Reply:
ok